### DISCUSSION TITLE
Webview/Stand-Alone-App?

### DISCUSSION THREAD

**Discussion by Clusters** [audioling/audioling#2] [Announcements]
With the switch to Tauri, the underlying Framework will change drastically. I just looked over Tauri and I assume audioling will be developed as an Stand-Alone-App only? Which was also the recommended way to use Feishin as well.
Am I correct?

**Comment by v0id-3r** [comment]
I use the feishin docker version because I can use it at work as well.
After countless searches for a webfront for Navidrome I found feishin and was really pleased on how polished it is and that is is working in web.
Would be a bummer if the switch would result in removal of the docker version :(

**Comment by jeffvli** [comment]
Tauri is just the desktop framework for the app. It's still being developed with web technologies (React) which will allow it to be deployed as a webapp so no worries there.

**Reply by Clusters** [reply]
So there might be a possible feature difference between Desktop and Webapp, like with Feishin?

**Reply by jeffvli** [reply]
The goal this time around is for feature-parity **except** for special cases such as:
- The player implementation -- since mpv isn't usable in the browser
- OS-level global hotkeys
- Anything else that requires interaction with the OS desktop

In audioling's case, it has a local http server that can handle desktop-only features that Feishin had, such as the external lyrics fetcher.

**Comment by hhenne1** [comment]
environment variables for a optional default/public user+password would be nice

**Comment by synchromatik** [comment]
One of the things that is a bit annoying in web feishin is playlist disappearing on page reload. It would be nice if web view of rewrite addressed this.

**Reply by v0id-3r** [reply]
This was something I submitted as suggestion for feishin, so I really hope this will be adressed! :)
